unicodetools/src/test/java/org/unicode/tools/emoji/unittest/TestEmojiData.java

```diff
@@ -595,37 +595,37 @@ public void TestBuckets(String type, EmojiDataSource emojiDataToTest, EmojiOrder
     @ParameterizedTest(name = "{0}")
     @MethodSource("dataProvider")
     public void TestAlphagram(String type, EmojiDataSource emojiDataToTest, EmojiOrder emojiOrder) {
-	    final EmojiData released = EmojiData.of(Emoji.VERSION_BETA);
-	    Multimap<String,String> toRgi = TreeMultimap.create();
-	    UnicodeSet discarded = new UnicodeSet();
-	    for (String item : released.getAllEmojiWithoutDefectives()) {
-		String alphagram = getAlphagram(item, released, discarded);
-		toRgi.put(alphagram, item);
-	    }
-	    int counter = 0;
-	    System.out.println();
-	    for ( Entry<String, Collection<String>> entry : toRgi.asMap().entrySet()) {
-		String key = entry.getKey();
-		Collection<String> val = entry.getValue();
-		if (val.size() > 1) {
-		    System.out.println(++counter + ") " + key + " => " + val);
-		}
-	    }
-	    System.out.println("Discarded: " + discarded);
+        final EmojiData released = EmojiData.of(Emoji.VERSION_BETA);
+        Multimap<String,String> toRgi = TreeMultimap.create();
+        UnicodeSet discarded = new UnicodeSet();
+        for (String item : released.getAllEmojiWithoutDefectives()) {
+        String alphagram = getAlphagram(item, released, discarded);
+        toRgi.put(alphagram, item);
+        }
```

Comment:
This file wanted 8sp/tab.

Reply:
Fixed in https://github.com/unicode-org/unicodetools/pull/178/commits/7c5c10bdc0232617b772ea6dd0e1e9389a3a9cfc